core/trino-main/pom.xml

```diff
@@ -535,6 +535,11 @@
         <resources>
             <resource>
                 <directory>src/main/resources</directory>
+                <includes>
```

Comment:
please drop it, and exclude instead to exclude sources

Reply:
done. It's using exclude only